app/services/analytics/base_service.rb

```diff
@@ -1,15 +1,36 @@
 # frozen_string_literal: true
 
+require "lago_http_client"
```

Comment:
How do you feel about making a new BaseService for analutics services calling the new data API? Could be in the same module/folder or even a separate one if you prefer.
I think it would help to differentiate the services that calls the external API and those using PG.


```ruby
module Analytics
  class RevenueStreamsService < BaseDataApiService   # <--- makes http calls
    # ...
  end
end

module Analytics
  class InvoicedUsagesService < BaseService   # <--- can be renamed too 
    # ...
  end
end
```

Reply:
yes - that makes sense - will do it 👍